app/controllers/packages_controller.rb

```diff
@@ -0,0 +1,78 @@
+class PackagesController < ApplicationController
+  skip_before_filter :verify_authenticity_token, only: [:save]
+
+  def estimate_request # api endpoint
+      @betsy_shipping = params
+      origin
+      destination
+      package
+
+      begin
+        ups = ActiveShipping::UPS.new(:login => ENV["ACTIVESHIPPING_UPS_LOGIN"],
+                                      :password => ENV["ACTIVESHIPPING_UPS_PASSWORD"],
+                                      :key => ENV["ACTIVESHIPPING_UPS_KEY"])
+        ups_response = ups.find_rates(@origin, @destination, @package)
+        ups_rates = ups_response.rates.sort_by(&:price).collect { |rate| [rate.service_name, rate.price, rate.delivery_date] }
+        usps = ActiveShipping::USPS.new(:login => ENV["ACTIVESHIPPING_USPS_LOGIN"])
+        usps_response = usps.find_rates(@origin, @destination, @package)
+        usps_rates = usps_response.rates.sort_by(&:price).collect { |rate| [rate.service_name, rate.price] }
```

Comment:
The `find_rates` and then `sort`...`collect` calls look very similar. Can you encapsulate those in a method?